addons/ai/functions/fnc_assignNVG.sqf

```diff
@@ -0,0 +1,39 @@
+#include "script_component.hpp"
+/*
+ * Author: Jonpas
+ * Assigns AI first found NVG in their inventory during night time and unassigns it during day time.
+ *
+ * Arguments:
+ * None
+
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call ace_ai_fnc_assignNVG
+ *
+ * Public: No
+ */
+
+if (!GVAR(assignNVG)) exitWith {};
```

Comment:
```suggestion
```
I think this check can be safely omitted.

Reply:
No, it's used to prevent any changes from class EH (unit spawning).